TL improvement suggestion (#368)
07th-mod/higurashi-console-arcs: Update/omot_018.txt

			NULL, "\"Thanks in advance, Coach.", Line_WaitForInput);
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600671", 256, TRUE);
	OutputLine(NULL, "とりあえず私を呼び出すときは、園崎姉妹のきれいな方って言ってくれたら、マッハで出ますので」",

> 			NULL, " And if you want to give me a call, please call for the more beautiful of the Sonozaki sisters, and I will come running at mach speed.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " And if you want to give me a call, please ask for the more beautiful of the Sonozaki sisters, and I'll come running at mach speed.\"", Line_Normal);
```

//r『先に行ってるわよ～！』とお母さんらしき人が笑顔で言うと、元気よく頷いて顔をこちらに戻した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "『先に行ってるわよ～！』とお母さんらしき人が笑顔で言うと、元気よく頷いて顔をこちらに戻した。",

> 			NULL, "The mother-like woman said, \"We'll go on ahead~!\" So Natsumi cheerfully nodded at the woman and turned her face back to us.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The mother-like woman said, \"We'll go on ahead~!\" So Natsumi cheerfully nodded at the woman and turned back to us.", Line_Normal);
```

			NULL, "\"...Hey, I had a thought.", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101481", 256, TRUE);
	OutputLine(NULL, "ここだと普通の女の子のほうが、浮いて見えるよな」",

> 			NULL, " Normal girls get out of place in this village, don't they?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Normal girls seem out of place around here, don't they?\"", Line_Normal);
```

	//rとりあえず、俺たちは総がかりになって暴走、いや暴発寸前のレナを抑え込み、ようやく女の子…美雪ちゃんの解放に成功。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "とりあえず、俺たちは総がかりになって暴走、いや暴発寸前のレナを抑え込み、ようやく女の子…美雪ちゃんの解放に成功。",

> 				NULL, "Rena was running wild, or maybe I should say about to explode, but we eventually managed to pin her down among us all... and set Miyuki-chan free.", Line_Normal);

Reviewer: ```suggestion
				NULL, "Rena was running wild, or maybe I should say about to explode, but between all of us we eventually managed to pin her down... and set Miyuki-chan free.", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 3, "ps3/s14/03/vt4bc_mion1006", 256, TRUE);
		OutputLine(NULL, "「しかし、あれだけの目に遭っておきながら無邪気というか、肝が据わっているというか…将来有望だね」",

> 				NULL, "\"At any rate, even though this girl had such a terrible experience, she is innocent enough, or better, has a lot of guts... So she'll have a promising future.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"At any rate, even though this girl has been through such a terrible experience, she's still so innocent. Or maybe I should say she has a lot of guts... So she'll have a promising future.\"", Line_Normal);
```

//r…まぁ、これが普通の反応だろう。しかし『世界あなた空気読めないで賞』のグランプリに輝く魅音にとっては、そんなことは些細なことでしかない！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…まぁ、これが普通の反応だろう。しかし『世界あなた空気読めないで賞』のグランプリに輝く魅音にとっては、そんなことは些細なことでしかない！",

> 			NULL, "...Well, I guess that's a normal reaction. But the grand prize \"unable to read the situation\" award still goes to Mion, so of course she never give a shit about it!", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Well, I guess that's a normal reaction. But the grand prize \"unable to read the situation\" award still goes to Mion, so of course that was a mere trifle to her!", Line_Normal);
```

//rそして、その目が語っている…『オ・マ・エ・ガ・カ・ワ・ル・カ？』と。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、その目が語っている…『オ・マ・エ・ガ・カ・ワ・ル・カ？』と。",

> 			NULL, "And those eyes were speaking to me... \"DO YOU WANNA STAND IN FOR HER?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "And those eyes were speaking to me... \"YOU WANNA TAKE HER PLACE?\"", Line_Normal);
```

	//rもっとも、そのせいで赤坂さんが迎えに来た時は『もっと一緒に遊ぶ～！』と逆に泣いてしまって、レナは見送りながら本気で涙ぐんでいたが…。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "もっとも、そのせいで赤坂さんが迎えに来た時は『もっと一緒に遊ぶ～！』と逆に泣いてしまって、レナは見送りながら本気で涙ぐんでいたが…。",

> 				NULL, "And when Akasaka-san came back to pick her up, she was seriously crying and shouting, \"Let's play together again~!\" And Rena saw her off between tears, too...", Line_Normal);

Reviewer: ```suggestion
				NULL, "And when Akasaka-san came back to pick her up, she was seriously crying and shouting, \"Let's play together again~!\" And Rena teared up seeing her off, too...", Line_Normal);
```

//rそんな感じで、いきなりがけっぷち状態から始まった、叩き売りオークション。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そんな感じで、いきなりがけっぷち状態から始まった、叩き売りオークション。",

> 			NULL, "As you can see, the auction required a last-ditch effort of me from the beginning.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Just like that, the auction began like standing on the edge of a cliff.", Line_Normal);
```

//rが、俺の掛け声と売り文句に見ている人たちが爆笑したり、興味津々に足を止めて覗き込んでくれるのがとても楽しくて、そこからはもう、普段の俺の本領発揮。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "が、俺の掛け声と売り文句に見ている人たちが爆笑したり、興味津々に足を止めて覗き込んでくれるのがとても楽しくて、そこからはもう、普段の俺の本領発揮。",

> 			NULL, "But it was very fun to see the people watching me shout my sales pitches were bursting into laughter and that they were so curious they had to stop and look. So I was able to get back into my usual flow.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But it was very fun to see the people watching me shout my sales pitches were bursting into laughter, and that they were so curious they had to stop and look. So I was able to get back into my usual flow.", Line_Normal);
```

//r追手はまぶしいほどのサーチライトで姿を見つけ出そうとしているが、素人技量もはなはだしい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "追手はまぶしいほどのサーチライトで姿を見つけ出そうとしているが、素人技量もはなはだしい。",
			NULL, "The pursuers were waving a searchlight around trying to find him, but his handiwork was far too intense for an amateur.", Line_Normal);
	ClearMessage();

//rそれだと逃げる相手は道を見つけやすくなるだけで、逆に追手は反射光が邪魔になる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それだと逃げる相手は道を見つけやすくなるだけで、逆に追手は反射光が邪魔になる。",
			NULL, "Thanks to that, it gave the fleeing man an easier time finding his way, but the reflected light also got in the pursuers' way.", Line_Normal);
	ClearMessage();

//r小此木が嘆くわけだ…そんなことを呟いて、場違いな感想を持った自分に苦笑した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "小此木が嘆くわけだ…そんなことを呟いて、場違いな感想を持った自分に苦笑した。",

> 			NULL, "\"That's why Okonogi sighs over them...\" He laughed at himself for thinking something so impertinent as he murmured that.", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"No wonder Okonogi is so distraught...\", he muttered to himself, then he laughed for thinking something so impertinent.", Line_Normal);
```

	ClearMessage();

//富竹rvS14/08/210800032.「……人のことは、言えた義理じゃないな…」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#945c44>富竹</color>", NULL, "<color=#945c44>Tomitake</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 8, "ps3/s14/08/210800032", 256, TRUE);
	OutputLine(NULL, "「……人のことは、言えた義理じゃないな…」",

> 			NULL, "\"...Who am I to say about them...?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...Who am I to say things like that about them...?\"", Line_Normal);
```

	ClearMessage();

//r自分とて鷹野さんと入江の関係にどっぷりとつかりすぎて、監視すべき本来の業務を忘れてしまっていたではないか。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "自分とて鷹野さんと入江の関係にどっぷりとつかりすぎて、監視すべき本来の業務を忘れてしまっていたではないか。",

> 			NULL, "He remembered he got too personally involved with Takano-san and Irie and that he forgot his original job was to monitor them.", Line_Normal);

Reviewer: ```suggestion
			NULL, "He recalled he got too personally involved with Takano-san and Irie, and that he forgot his original job was to monitor them.", Line_Normal);
```

//rそして、出来上がったものは……この、雛見沢全体の『破滅』。壮大な、鷹野三四が描き出した神に奉納する『祭』の姿だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、出来上がったものは……この、雛見沢全体の『破滅』。壮大な、鷹野三四が描き出した神に奉納する『祭』の姿だった。",

> 			NULL, "And the result of that... is a \"catastrophe\" that will wipe out all of Hinamizawa. It's a magnificent \"ritual\" dedicated to the god portrayed by Miyo Takano.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And the result of that... is a \"catastrophe\" that will wipe out all of Hinamizawa. It's a magnificent \"ritual\" dedicated to God, as depicted by Miyo Takano.", Line_Normal);
```

//r誰も見ていないすきに祭具殿の中身が見たい、と鷹野さんに誘われた私は、二人連れ立って古手神社の境内奥に足を運んだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "誰も見ていないすきに祭具殿の中身が見たい、と鷹野さんに誘われた私は、二人連れ立って古手神社の境内奥に足を運んだ。",

> 			NULL, "Takano-san invited me to see the inside of the ritual storehouse surreptitiously, so I went to the recesses of the Furude Shrine grounds with her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Takano-san surreptitiously invited me to see the inside of the ritual storehouse, so I went to the recesses of the Furude Shrine grounds with her.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "髪をかき上げた鷹野さんに流し目を向けられ、思わず息を呑む。…その仕草が、私をどこか見下し、そして挑発するものだとわかってはいたが、…その妖しい雰囲気が彼女に惹かれた理由でもあった。",

> 			NULL, "Takano-san turned a sideways glance this way while sweeping her hair back, so I unintentionally caught my breath. ...I could tell she was looking down on me and trying to provoke me... but that mysterious aura was what attracted me to her in the first place.", Line_Normal);

Reviewer: ```suggestion
			NULL, "As Takano-san turned a sideways glance this way, sweeping her hair back, I suddenly caught my breath. ...I could tell she was looking down on me and trying to provoke me... but that mysterious aura was what attracted me to her in the first place.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f6d9a8>鷹野</color>", NULL, "<color=#f6d9a8>Takano</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 9, "ps3/s14/09/210900107", 256, TRUE);
	OutputLine(NULL, "「それでも、…ひたすら人の善意を信じて、いつかは報われると身を削る思いを重ねてきた私の気持ちを、",

> 			NULL, "\"But even so... I always believed in the goodwill of others and had a pile of tough times so that someday I'd be rewarded.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"But even so... I always believed in the goodwill of others, and underwent great hardship hoping that someday I'd be rewarded.", Line_WaitForInput);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#3e5745>小此木</color>", NULL, "<color=#3e5745>Okonogi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 14, "ps3/s14/14/213400008", 256, TRUE);
		OutputLine(NULL, "「…………ターゲット、電気銃で狙撃。",

> 				NULL, "\"...Target successfully tasered by sniper fire.", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"...Target successfully tased by sniper fire.", Line_WaitForInput);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#3e5745>小此木</color>", NULL, "<color=#3e5745>Okonogi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 14, "ps3/s14/14/213400012", 256, TRUE);
		OutputLine(NULL, "「だぁほっ！",

> 				NULL, "\"Y'all fucking idiots!", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"Seize him!", Line_WaitForInput);
```

Author: だぁほ is corrupted form of どあほ or どアホ.

Reviewer: Are you sure it's not 拿捕?

Author: Yes. If it's 拿捕, he'll say like 拿捕せぇ! with kanji.

		ModPlayVoiceLS(4, 14, "ps3/s14/14/213400013", 256, TRUE);
		OutputLine(NULL, "　目標を見つけるまで全員、任務は継続に決まっとぉが！",
				NULL, " All hands, continue your mission until the target has been found!", Line_WaitForInput);
		ModPlayVoiceLS(4, 14, "ps3/s14/14/213400014", 256, TRUE);
		OutputLine(NULL, "　何のための捜索隊ね、すっとぉしゃんとせっかぁ！！」",

> 				NULL, " Try to understand the search party's goal! Use your brain!!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " What is this search party even for? Use your brains!!\"", Line_Normal);
```

			NULL, "\"...If we move around too much, it might cause Takano and the others to expedite their plans.", Line_WaitForInput);
	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500923", 256, TRUE);
	OutputLine(NULL, "消極的かもしれませんが、今は赤坂と富竹の動きを待つときだと思うのですよ」",

> 			NULL, " Even so we're too passive, for now I think we should wait on Akasaka and Tomitake before taking action.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " This may be excessively passive, but for now I think we should wait on Akasaka and Tomitake before taking action.\"", Line_Normal);
```

				NULL, "\"Well, we'll have to make her work for the ones that were duds.", Line_WaitForInput);
		ModPlayVoiceLS(4, 0, "ps2/00/villc11006", 540, TRUE);
		OutputLine(NULL, "神輿に乗せておだてておけば、少しは恩恵にあずかれるかと思ってたんだが」",

> 				NULL, " Because we expected that we'd benefit from her if she played leader leader and got selfcomplacent.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Because we thought we'd benefit from her if she played leader and we flattered her.\"", Line_Normal);
```

		ClearMessage();

		DrawScene("black", 1000 );

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#8676cf>羽入</color>", NULL, "<color=#8676cf>Hanyuu</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 12, "ps2/12/221100028", 540, TRUE);
		OutputLine(NULL, "神輿…………おだてて…………だと？！",

> 				NULL, "Playing leader... and self... complacent?!", Line_Normal);

Reviewer: ```suggestion
				NULL, "Playing leader... and... flattered her?!", Line_Normal);
```
